{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/templates/404.html

```diff
@@ -5,5 +5,5 @@
 {% block content %}
 <h1>Page not found</h1>
 
-<p>This is not the page you were looking for.</p>
+<p>{% if exception %}{{ exception }}{% else %}This is not the page you were looking for.{% endif %}</p>
```

Comment:
If your view raises `Http404()`, the default `This is not the page you were looking for` message is displayed, right?

Reply:
Yup!